sw/device/examples/hello_usbdev/meson.build

```diff
@@ -5,7 +5,9 @@
 foreach device_name, device_lib : sw_lib_arch_core_devices
   hello_usbdev_elf = executable(
     'hello_usbdev_' + device_name,
-    sources: ['hello_usbdev.c'],
+    sources: [
```

Comment:
Why this change? Isn't it supposed to be on a single line when there is only a single element inside the brackets?

Reply:
Doing it this way makes it a lot easier to add other sources or dependencies (future changes will have fewer diff lines). I'm not sure we have a meson style guide at the moment.

Reply:
I guess we don't need a style guide for everything. Keeping things on separate lines and with a trailing comma is good practice to keep diffs small and reduce the probability of merge conflicts (independent of the language).

Reply:
Not a fan of the new style, but I think Sam's justification is pretty solid, as far as git greping for stuff goes (I don't think it'll be as helpful with merge conflicts but w/e).